### PR TITLE
#34898 Submit task with file upload

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+// @ts-ignore
```
Comment (Rick-Ritense) [inline packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx:3]: Why did you need the ts-ignore?

File: packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  constructor(component: any, options: any, data: any) {
+    super(component, options, data);
+    this.component = component;
+
+    if (this.component.multipleFiles === undefined) {
+      this.component.multipleFiles = true;
+    }
+
+    this.component.multiple = true; // Must be true to force formio to accept arrays as valid input value for this field type
```
Comment (Rick-Ritense) [inline packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx:19]: Does linting allow comments behind code?

**Comment by Rick-Ritense** [comment]
I miss license headers in all new files

File: packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx (hunk 1 above)
Comment (Klaas-Ritense) [reply packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx:3]: Its a bug fix. Its necessary because the '@formio/react' library doesn't have typing. If that line is removed, it will throw error 'TS7016'

File: packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx (hunk 2 above)
Comment (Klaas-Ritense) [reply packages/user-interface/src/components/form-io-uploader/form-io-uploader.tsx:19]: It is very strict, but I just checked it and this is allowed

**Review by Rick-Ritense — APPROVED** [review]
